docs/source/markdown/podman-run.1.md

```diff
@@ -901,17 +901,19 @@ Ulimit options. You can use **host** to copy the current configuration from the
 
 Sets the username or UID used and optionally the groupname or GID for the specified command.
 
-Without this argument the command will be run as root in the container.
+Without this argument the command will be run as root in the container, unless a USER has been
```

Comment:
Maybe this should read "Without this argument the command will be run as root on the host unless a user was specified by the container image or the **--user** option"

Reply:
Not root on the host, since it could be rootless or running in a user namespace. It will also run as the User of the image.  So the wording of this should be

Without this argument the command will run as the user specified in the container image.  Note that this user often defaults to root.  The actual UID of the process running in the container is based on the User namespace the container is running in.  By default if you are running a rootful Podman, the container will run as the hosts root.  If you are running the container as a rootless user, the root inside of the container is actually the users UID.

Reply:
I think it's important to get the UESR/--user option thought in.  So I've tweaked the comment from @rhatdan to:
```
Without this argument, the command will run as the user specified in the container image. Unless overridden by a `USER` command in the Containerfile or by a value passed to this option, this user generally defaults to root. The actual UID of the process running in the container is based on the User namespace the container is running in. By default, if you are running a rootful Podman, the container will run using the UID of the host's root. If you are running the container as a rootless user, by default the UID used by root inside of the container is the rootless user's UID.
```